linera-views/src/backends/rocks_db.rs

```diff
@@ -45,21 +44,228 @@ const MAX_KEY_SIZE: usize = 8388208;
 /// The RocksDB client that we use.
 pub type DB = rocksdb::DBWithThreadMode<rocksdb::MultiThreaded>;
 
+/// The choice of the spawning mode.
+/// The SpawnBlocking always works and is the safest.
+/// The BlockInPlace can only be used in multi-threaded environment.
```

Comment:
"The SpawnBlocking" -> "`SpawnBlocking`"

Reply:
Ok.